testonly/hasher.go

```diff
@@ -17,7 +17,16 @@ package testonly
 // This file implements the hashing functions that are part of a Trillian
 // personality.
 
-import "crypto/sha256"
+import (
+	"crypto"
+	"crypto/sha256"
+
+	"github.com/google/trillian/merkle/rfc6962"
+)
+
+// Hasher is the default hasher for tests.
+// TODO: Make this a custom algorithm to decouple hashing from coded defaults.
```

Comment:
I don't think we can do this very soon.  Doing this means re-writing the tests such that they don't depend on test vectors that are specific to this hash function. Anyone want to volunteer?